### ISSUE TITLE
Vulnerable dependency

### ISSUE THREAD

**Issue by ghusse** [aacerox/node-rest-client#105]
I seems that `node-rest-client` depends on `ms 0.7.0` which is identified by node security project as vulnerable.

I'm running a continuous integration task every day on our dependencies (and node-rest-client is one of them), to check vulnerabilities with NSP (node security project).

Our build fails because of this dependency. Here is the report:

```
(+) 1 vulnerabilities found
┌───────────────┬───────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                  │
├───────────────┼───────────────────────────────────────────────────────┤
│ Name          │ ms                                                    │
├───────────────┼───────────────────────────────────────────────────────┤
│ Installed     │ 0.7.0                                                 │
├───────────────┼───────────────────────────────────────────────────────┤
│ Vulnerable    │ <=0.7.0                                               │
├───────────────┼───────────────────────────────────────────────────────┤
│ Patched       │ >0.7.0                                                │
├───────────────┼───────────────────────────────────────────────────────┤
│ Path          │ node-rest-client > debug > ms                         │
├───────────────┼───────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/46                 │
└───────────────┴───────────────────────────────────────────────────────┘
```

Can you please update your dependency?

**Comment by ghusse** [comment]
(issue updated to include the report)

**Comment by aacerox** [comment]
fixed, I have update debug to 2.2.0 to solve this issue.

**Comment by ghusse** [comment]
Thanks
